specs/pvs/corla.pvs

```diff
@@ -132,14 +132,21 @@
 fafessl: THEORY
```

Comment:
Yes, given recent information from CDOS, this should be refined to a list (which is a finite sequence).